src/github.com/getlantern/lantern-android/app/src/org/getlantern/Flashlight.java

```diff
@@ -12,9 +12,9 @@ private Flashlight() {} // uninstantiable
     public static void RunClientProxy(String listenAddr) throws Exception {
```

Comment:
This file is generated by the build process, we should probably not take it into the source tree as it is not really editable.